### PR TITLE
Added adapters for Omniguards & Eternals

### PR DESCRIPTION
**Pull request by CallReef**
Signed-off by: CallReef <bullishgem@gmail.com>

### CODE REVIEW

File: component.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
+// Copyright Abridged, Inc. 2022. All Rights Reserved.
```
Comment (raymondfeng) [inline component.ts:1]: Hmm, please move all files to `src` directory.

Comment (CallReef) [reply component.ts]: No problem. All moved over

**Comment by raymondfeng** [comment]
@CallReef There are a few issues in the pull request. I have pushed a fix to your repo - thus this PR.

**Comment by raymondfeng** [comment]
@CallReef The 3rd commit you pushed completely messed up everything and it does not even compile. Please run `npm run build` before pushing.

I understand that your want to have on adapter to support multiple staked assets on the same staking contract. I'll force push a new commit.

**Pull request merged**